### PR TITLE
[PR 1] DEV-178 add state orders

### PR DESCRIPTION
**Pull request by jmaralo**
This PR adds the logic to decode the state orders message and send it to the frontend. The state orders are special orders that are not always enabled, and are instead enabled and disabled by the boards with the special state orders message, which is a list of enabled orders for a specific board.

There is a new kind of packet in the ADE, the stateOrders.

Since @smorsua was working on some changes for the frontend communication, I needed to merge his branch into mine, only changes to files inside `vehicle` and `order_transfer` are relevant, the rest are the same ones as in DEV-181

### CODE REVIEW

**Comment by jmaralo** [comment]
Wait until DEV-181 is merged to develop before merging this PR to avoid any conflicts that might arise

**Review by smorsua — APPROVED** [review]

**Review by lipezaballa — APPROVED** [review]

**Pull request merged**
